### PR TITLE
Various UI improvements

### PR DESCRIPTION
**Pull request by evguu**
(I'll open this as a draft while I make it so I can compare easier)

### CODE REVIEW

**Comment by evguu** [comment]
Typescript doesn't raise warnings for absolute imports anymore! You can stop using '../../../..../../../../../../' monsters now :D

**Comment by evguu** [comment]
Also this PR makes sure that warning text is displayed both in the top and bottom of the page when it's applicable

**Comment by evguu** [comment]
I mean, I didn't even touch anything related to models :smiling_face_with_tear:

**Comment by evguu** [comment]
My guess is that it's the same problem that I've had with worker list, in which case everything should run fine on your side

File: pages/index.tsx [hunk 1]
```diff
@@ -677,21 +678,13 @@ const Home: NextPage = ({ modelDetails, shortlinkImageParams }: any) => {
...
-          <div className="mt-2 text-red-500 font-semibold">
-            Error: {hasError}
-          </div>
-        )}
+
+        <FormErrorMessage
+          hasValidationError={hasValidationError}
+          hasError={hasError}
+          fixedSeedErrorMsg={fixedSeedErrorMsg}
+        />
```
Comment (daveschumaker) [inline pages/index.tsx:689]: (Bear with me as I step through this commit by commit).

I like this idea of creating an specific error component. This particular feedback is more for me, but those props make me think that I should really standardize the error messages the web app is generating. Will add to the to-do list and definitely not a blocker for the PR.

Comment (daveschumaker) [reply pages/index.tsx:689]: Oh, GitHub. Now that I view this in the files changed panel, this comment is attached nowhere close to where I was commenting on. 🙃

File: tsconfig.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": ".",
```
Comment (daveschumaker) [inline tsconfig.json:3]: HAH! Thank you! Much nicer dev experience. 😅

**Review by daveschumaker — APPROVED** [review]
Sweet! Nice job with this. A lot of great improvements here.

**Pull request merged**